dlx_rest/static/js/validation.js

```diff
@@ -875,6 +875,26 @@ export const validationData = {
             "validSubfields": ["a", "g", "t"],
             "defaultSubfields": ["a"] 
         },
+        "506": {
+            "name": "Access restriction note",
+            "required": false,
+            "repeatable": false,
+            "validIndicators1": [],
+            "validIndicators2": [],
+            "requiredSubfields": [],
+            "validSubfields": ["a", "b", "f", "g"],
+            "defaultSubfields": ["a", "b", "f"],
+            "validStrings": {
+                "a": ["Restricted distribution","Confidential","Embargoed","Withdrawn","De-restricted"],
+                "f": ["Access limited to authorized users", "Access limited on-site", "Access limited on-site and to authorized users"]
+            },
+            "_validRegex": {
```

Comment:
Note that the underscore in "_validRegex" acts as a comment here. [The field name recognized in the validation checker is "validRegex" (no underscore)](https://github.com/dag-hammarskjold-library/dlx-rest/blob/3f294b4d6127e29f5b09d9cc52aa6d71f1c04904/dlx_rest/static/js/jmarc.mjs#L129).

Reply:
Just to clarify though: do I need both validRegex and isDate if the isDate flag is validating the date format?